internal/cmd/util/exit.go

```diff
@@ -23,7 +23,10 @@ import (
 
 // SetExitCode sets the exit code to 1 if the error is not a context.Canceled error.
 func SetExitCode(err error) {
-	if (err != nil) && !errors.Is(err, context.Canceled) {
+	if (err != nil) && errors.Is(err, context.DeadlineExceeded) {
+		errorExitCodeChannel <- 124 // Indicate that there was a timeout error
```

Comment:
If err is a DeadlineExceeded, we want to set the exit code to 124 instead of 1.
This will be used here: https://github.com/cert-manager/cert-manager/pull/6119/files#diff-cfd76d5765e1105fc331f3c440d29ad4a9939557e16706b887785d24c13626e6R138

Reply:
I wish this type of information was documented in the man pages, but we don't have a man page for `cmctl`...